sdk/communication/communication-chat/src/generated/src/models/parameters.ts

```diff
@@ -183,7 +183,7 @@ export const createChatThreadRequest: OperationParameter = {
 export const repeatabilityRequestId: OperationParameter = {
   parameterPath: ["options", "repeatabilityRequestId"],
   mapper: {
-    serializedName: "repeatability-request-id",
+    serializedName: "repeatability-Request-Id",
```

Comment:
@amrelroumy-msft is going to fix this in swagger. Will revert this change after it.